digital_comms/ccam.py

```diff
@@ -117,7 +201,16 @@ def system(self):
         return system
 
     def capacity(self):
-        """Return the mean capacity from all nested postcode sectors
+        """Calculate capacity as the mean capacity from all nested postcode sectors
```

Comment:
Perhaps this should just be 'Calculate mean capacity from all nested postcode sectors' for simplicity

Reply:
Done